README.md

```diff
@@ -75,9 +82,11 @@ Select 1:
 Identify the three places CSS can go, and rank them in terms of specificity:
 
 ```text
-1.
-2.
-3.
+1. MOST SPECIFICITY - inside an html elements tag e.g. <p styel="color:blue;">Example</p>
+
+2. CAN BE SPECIFIC - between style tags in the html file e.g. <style>p{color:blue;}</style>
+
+3. CAN BE SPECIFIC - link to a separate file e.g. <link rel="stylesheet" href="style.css"/>
```

Comment:
:+1: although, to be explicit, the external stylesheet is the least specific of the three.